frontend/src/pages/Dashboard/index.tsx

```diff
@@ -119,28 +120,6 @@ const Dashboard: React.FC = () => {
     );
   }
 
-  if (classroomUser?.classroom_role === "STUDENT") {
-    return (
-      <div className="Dashboard__unauthorized">
-        <h2>Access Denied</h2>
-        <p>
-          You do not have permission to view the classroom management dashboard.
```

Comment:
For some reason, when I try to view the dashboard for a class as a student, it navigates me back to org select, and does not show me this access denied page

Reply:
fixed.